lms/djangoapps/courseware/module_render.py

```diff
@@ -162,11 +174,11 @@ def toc_for_course(request, course, active_chapter, active_section, field_data_c
                                      'active': active,
                                      'graded': section.graded,
                                      })
-            chapters.append({'display_name': chapter.display_name_with_default,
-                             'url_name': chapter.url_name,
-                             'sections': sections,
-                             'active': chapter.url_name == active_chapter})
-        return chapters
+            toc_chapters.append({'display_name': chapter.display_name_with_default,
```

Comment:
Optional: I'm pretty sure we've standardized breaking long lines a la [this doc](https://github.com/edx/edx-platform/wiki/Python-Guidelines).